src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingWorkCoordinator.cs

```diff
@@ -21,12 +21,15 @@ internal partial class UnitTestingSolutionCrawlerRegistrationService
     {
         internal sealed partial class UnitTestingWorkCoordinator : IUnitTestingWorkCoordinator
         {
-            private readonly UnitTestingRegistration _registration;
+#if false // Not used in unit testing crawling
             private readonly object _gate = new();
+            private readonly IUnitTestingDocumentTrackingService _documentTrackingService;
```

Comment:
Is this interface entirely unused and should it be commented out too?

Reply:
the interface still seems to be used for *other* purposes.  For example, prioritizing the visible/open documents.  it's just not needed to know when the active doc switches.